src/main/java/com/rocketden/main/dto/room/RoomMapper.java

```diff
@@ -9,12 +9,11 @@ protected RoomMapper() {}
 
     private static final ModelMapper mapper = new ModelMapper();
 
-    public static CreateRoomResponse entityToCreateResponse(Room entity) {
-        return mapper.map(entity, CreateRoomResponse.class);
-    }
-
-    public static JoinRoomResponse entityToJoinResponse(Room entity) {
-        return mapper.map(entity, JoinRoomResponse.class);
+    public static RoomDto toDto(Room entity) {
+        if (entity == null) {
+            return null;
+        }
+        return mapper.map(entity, RoomDto.class);
     }
 
     public static GetRoomResponse entityToGetResponse(Room entity) {
```

Comment:
Do you still need this method? Or can you update it to use `RoomDto`?

Reply:
On that note, did you leave in the `GetRoomResponse` file on purpose?

Reply:
Yeah, figured that the tradeoffs were keeping the separate get response means that you don't need to return all the data (users, settings, etc.) if you're just validating that a room exists, while at the same time that would mean you need to have a separate class specifically for the GET endpoint. So I'm okay with doing this either way, just lmk

Reply:
Ok that makes sense, no need to update -- thanks for the explanation.